src/partials/header.html

```diff
@@ -0,0 +1,35 @@
+<header class="header__line">
+    <a class="logo" href="index.html" title="Main page">
+        <svg class="logo__icon" width="35.28" height="42.8">
+            <use href="#"></use>
+          </svg>
+        <svg class="logo__icon" width="40.96" height="42.8">
+            <use href="#"></use>
+        </svg>
+        <svg class="logo__icon" width="40.96" height="42.8">
+            <use href="#"></use>
+        </svg>      
+    </a>
```

Comment:
druga sprawa logo traktujemy jako jeden plik svg dlatego będzie to zrobione w ten sposób
a class logo
svg width 45 heigth 35
use
i tyle, a później robimy tak samo z tel i hamburgerem